web/public/mgmt/config.yml

```diff
@@ -3025,6 +3051,32 @@ collections:
       - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
       - { label: "Filename (disabled after file creation)", name: "filename", widget: "write-once-read-only" }
       - { label: "Name/Title", name: "name", widget: "string" }
+      - {
+          label: "Apply to all - overrides industry and sector dropdown",
```

Comment:
[dust] Capitalize some of these labels: change to "Apply to All - Overrides Industry and Sector Dropdown"